src/app/content/highlights/hooks/closeMyHighlights.ts

```diff
@@ -0,0 +1,21 @@
+import { push } from '../../../navigation/actions';
+import { query } from '../../../navigation/selectors';
```

Comment:
i think the code reads more nicely if you do `* as navigation` or `* as selectNavigation` here, so that in the code its `selectNavigation.query` instead of just `query`

not blocking